modules/deploy/pages/redpanda/manual/resilience/shadowing-guide.adoc

```diff
@@ -0,0 +1,208 @@
+
+[IMPORTANT]
+====
+This is an emergency procedure. For planned failover testing or regular shadow link management, see xref:deploy:redpanda/manual/resilience/shadowing.adoc[]. Ensure you have completed the xref:deploy:redpanda/manual/resilience/shadowing.adoc#disaster-readiness-checklist[disaster readiness checklist] before an emergency occurs.
+====
+
+== Emergency failover procedure
+
+Follow these steps in order during an active disaster:
+
+1. <<assess-situation,Assess the situation>>
+2. <<verify-shadow-status,Verify shadow cluster status>>  
+3. <<document-state,Document current state>>
+4. <<initiate-failover,Initiate failover>>
+5. <<monitor-progress,Monitor failover progress>>
+6. <<update-applications,Update application configuration>>
+7. <<verify-functionality,Verify application functionality>>
+8. <<cleanup-stabilize,Clean up and stabilize>>
+
+[[assess-situation]]
+=== Assess the situation
+
+Confirm that failover is necessary:
+
+[,bash]
+----
+# Check if the primary cluster is responding
+rpk cluster info --brokers <primary-cluster-brokers>
+
+# If primary cluster is down, check shadow cluster health
+rpk cluster info --brokers <shadow-cluster-brokers>
+----
+
+**Decision point**: If the primary cluster is responsive, consider whether failover is actually needed. Partial outages may not require full disaster recovery.
+
+[[verify-shadow-status]]
+=== Verify shadow cluster status
+
+Check the health of your shadow links:
+
+[,bash]
+----
+# List all shadow links
+rpk shadow list
+
+# Check status of your disaster recovery link
+rpk shadow status <disaster-recovery-link-name>
+----
+
+Verify these conditions before proceeding with failover:
+
+* Shadow link state should be `ACTIVE`.
+* Topics should be in `ACTIVE` state (not `FAULTED`).
+* Replication lag should be reasonable for your RPO requirements.
+
+[[document-state]]
+=== Document current state
+
+Record the current lag and status before proceeding:
+
+[,bash]
+----
+# Capture current status for post-mortem analysis
+rpk shadow status <disaster-recovery-link-name> > failover-status-$(date +%Y%m%d-%H%M%S).log
+----
+
+IMPORTANT: Note the replication lag to estimate potential data loss.
+
+[[initiate-failover]]
+=== Initiate failover
+
+For complete cluster failover (recommended during disasters):
+
+[,bash]
+----
+# Fail over all topics in the shadow link
+rpk shadow failover <disaster-recovery-link-name> --all --no-confirm
+----
+
+For selective topic failover (if only specific services are affected):
+
+[,bash]
+----
+# Fail over individual topics
+rpk shadow failover <disaster-recovery-link-name> --topic <critical-topic-1> --no-confirm
+rpk shadow failover <disaster-recovery-link-name> --topic <critical-topic-2> --no-confirm
+----
+
+[[monitor-progress]]
+=== Monitor failover progress
+
+Track the promotion process:
+
+[,bash]
+----
+# Monitor status until all topics show PROMOTED
+watch -n 5 "rpk shadow status <disaster-recovery-link-name>"
+----
```

Comment:
Won't mark in each step, but setting up a specific example throughout this workflow and then showing all the data that matches that example would be a nice user experience with docs--give users the validation they are likely seeking during a potentially stressful situation.